src/macaron/errors.py

```diff
@@ -22,3 +22,11 @@ class CUEExpectationError(MacaronError):
 
 class CUERuntimeError(MacaronError):
     """Happens when there are errors in CUE expectation validation."""
+
+
+class ConfigurationError(MacaronError):
+    """Happens when there is an error in the configuration (.ini) file."""
+
+
+class CloneError(MacaronError):
+    """Happens when cannot clone a git repository (.ini) file."""
```

Comment:
I think `(.ini) file` should be removed?

Reply:
Thanks. Fixed in 8d642231b291181f84e666783d8a26de4061a59c.